Mathlib/SetTheory/Ordinal/Notation.lean

```diff
@@ -119,18 +119,25 @@ def ofNat (n : ℕ) : ONote := n
 @[simp] theorem natCast_succ (n : ℕ) : n.succ = oadd 0 n.succPNat 0 := rfl
 theorem natCast_one : (1 : ℕ) = (1 : ONote) := rfl
 
-instance (n : ℕ) : OfNat ONote n where
-  ofNat := n
-
 @[deprecated (since := "2024-10-17")] alias ofNat_zero := natCast_zero
 @[deprecated (since := "2024-10-17")] alias ofNat_one := natCast_one
 @[deprecated (since := "2024-10-17")] alias ofNat_succ := natCast_succ
 
 @[simp] theorem repr_natCast (n : ℕ) : repr n = n := by cases n <;> simp
-theorem repr_one : repr 1 = 1 := mod_cast repr_natCast 1
 theorem repr_zero : repr 0 = 0 := repr_natCast 0
+theorem repr_one : repr 1 = 1 := mod_cast repr_natCast 1
 
-@[deprecated (since := "2024-10-17")] alias repr_ofNat := repr_natCast
+instance (n : ℕ) : OfNat ONote n where
+  ofNat := n
+
+@[simp]
+theorem natCast_ofNat (n : ℕ) [n.AtLeastTwo] :
+    (no_index (OfNat.ofNat n) : ONote) = OfNat.ofNat n :=
```

Comment:
This lemma is a tautology

Reply:
(it also doesn't mention a cast; did you mean for one side to be case to `Ordinal`?

Reply:
What type do you intend the RHS to be?

Reply:
Maybe you meant
```suggestion
    ↑(no_index (OfNat.ofNat n) : ℕ) = (OfNat.ofNat n : ONote) :=
```